src/directives/ec-tooltip/ec-tooltip.story.js

```diff
@@ -15,37 +15,68 @@ const placement = 'top';
 const delay = 100;
 const classes = ['ec-tooltip--level-1'];
 
+const tooltipConfig = {
+  content: '<p>This is the normal tooltip</p>',
+  placement: 'bottom',
+  show: true,
+};
+
+const whiteTooltipConfig = {
+  content: '<p>This is a popover style tooltip</p>',
+  classes: ['ec-tooltip--bg-bright'],
+  trigger: 'click',
+  show: true,
+  placement: 'bottom',
+};
+
+
 const stories = storiesOf('Tooltip', module);
 
-stories.add('basic', () => ({
-  directives: { EcTooltip },
-  props: {
-    content: {
-      default: text('content', content),
-    },
-    placement: {
-      default: select('placement', ['top', 'bottom', 'left', 'right'], placement),
+stories
+  .add('basic', () => ({
```

Comment:
The ticket says: `Add all versions in a single story for regression tests`. Please modify this story  and add tooltips for all 4 positions.